processor/filterprocessor/config.go

```diff
@@ -36,6 +36,7 @@ type Config struct {
 
 	Logs LogFilters `mapstructure:"logs"`
 
+	// Deprecated: [v0.74.0] use TraceFilters instead
```

Comment:
This (and the other deprecation notices below) should say `v0.72.0`. From [the contributing docs][1]:

> Deprecation notice SHOULD contain **a version starting from which the deprecation takes effect** for tracking purposes.

https://github.com/open-telemetry/opentelemetry-collector/blob/main/CONTRIBUTING.md#breaking-changes

Reply:
Done